ddtrace/appsec/_metrics.py

```diff
@@ -81,12 +116,17 @@ def _set_waf_request_metrics(*args):
             # TODO: enable it when Telemetry intake accepts this tag
             # is_truncation = any((result.truncation for result in list_results))
 
+            truncation = result["truncation"]
+            input_truncated = bool(
+                truncation["string_length"] or truncation["container_size"] or truncation["container_depth"]
+            )
             tags_request = (
                 ("event_rules_version", result["version"]),
                 ("waf_version", DDWAF_VERSION),
-                ("rule_triggered", str(result["triggered"]).lower()),
-                ("request_blocked", str(result["blocked"]).lower()),
-                ("waf_timeout", str(result["timeout"]).lower()),
+                ("rule_triggered", bool_str[result["triggered"]]),
+                ("request_blocked", bool_str[result["blocked"]]),
+                ("waf_timeout", bool_str[result["timeout"]]),
+                ("input_truncated", bool_str[input_truncated]),
```

Comment:
mmmm, I don't know if that is a smart solution use a boolean as integers 0 and 1 or it's a madness, IMHO, I prefer bools with bools and integers with integers :sweat_smile: